### ISSUE TITLE
all protocols.io links should link to latest version of the protocol

### ISSUE THREAD

**Issue by ngehlenborg** [hubmapconsortium/portal-ui#739]
Related to #694 and other alpha testing feedback. 

This affects:

- [ ] sample metadata tables (once implemented)
- [ ] assay metadata tables
- [ ] donor protocols section
- [ ] sample protocols section

Per protocols.io help desk, links to the latest version can be generated like this:

```
For the last version link please use the query parameter latest_version  - https://www.protocols.io/view/vu-biomolecular-multimodal-imaging-center-biomic-k-bfskjncw?latest_version
```

I am not sure if this also works via the doi.org redirect. Please test and report back if that is *not possible*.

**Comment by ngehlenborg** [comment]
Hm, it looks like this doesn't work if you only have the DOI for a protocol. I will follow up with the helpdesk.

**Comment by mccalluc** [comment]
Nastia Malochka, on 2020-7-21:
> Hi Jonathan and Nils,
>
> I just wanted to let you know that we've added an optional parameter for the API last_version=1. It returns the data of the latest version of the protocol. 
>
>Please let me know if you have any questions.

**Comment by mccalluc** [comment]
API working now: https://www.protocols.io/api/v3/protocols/7hhhj36?last_version=1

**Comment by john-conroy** [comment]
Closed as of #896 .
